tests/integration/pilot/cni/cni_test.go

```diff
@@ -65,6 +65,10 @@ func TestCNIReachability(t *testing.T) {
 					ConfigFile: "global-mtls-on.yaml",
 					Namespace:  systemNM,
 					Include: func(src echo.Instance, opts echo.CallOptions) bool {
+						// Exclude headless naked service
```

Comment:
Can you comment why we are excluding it? Its clear what we are doing here, but not why (to people without context - i understand it from the PR)